yourls/yourls.php

```diff
@@ -86,7 +84,7 @@ function yourls_addon_settings(&$a,&$s) {
 
 }
 
-function yourls_addon_settings_post(&$a,&$b) {
+function yourls_addon_settings_post(App $a, array &$b) {
 
 	if(! is_site_admin())
```

Comment:
There are lots more code parts that lacks curly braces. Should be done in next PR.